client/go/outline/tun2socks/tunnel.go

```diff
@@ -26,6 +26,7 @@ import (
 
 	"github.com/Jigsaw-Code/outline-sdk/transport"
 
+	"github.com/Jigsaw-Code/outline-apps/client/go/outline" // Import the package containing Client
```

Comment:
No need to comment on this, `gofmt` will do the trick to maintain the used packages.

```suggestion
	"github.com/Jigsaw-Code/outline-apps/client/go/outline"
```

Reply:
Removed the comment.